dev/README_RELEASE_AIRFLOW.md

```diff
@@ -802,10 +802,11 @@ Documentation for providers can be found in the ``/docs/apache-airflow`` directo
     ```
 
 - Copy the documentation to the ``airflow-site`` repository, create commit, push changes, open a PR and merge it when the build is green.
+- Either provide the ``--airflow-site-directory`` flag or set the env variable: ``AIRFLOW_SITE_DIRECTORY``
```

Comment:
This is not needed. The `AIRFLOW_SITE_DIRECTORY` is already set in line 788. so this is not something optional - we already KNOW it has been set.

Same in all other cases.